src/cmap/command_monitoring_events.ts

```diff
@@ -117,7 +137,13 @@ export class CommandSucceededEvent {
  */
 export class CommandFailedEvent {
   address: string;
+  /** Driver generated connection id */
   connectionId?: string | number;
+  /**
+   * Server generated connection id
+   * Distinct from the connection id and is returned by the hello or legacy hello response as "connectionId" from the server on 4.2+.
+   */
+  serverConnectionId?: bigint;
```

Comment:
```suggestion
  serverConnectionId: bigint;
```

Same goes for the other event classes

Reply:
Sounds good, just implemented this change.